File: app/src/main/java/com/dinodevs/greatfitwatchface/theme/bin/Distance.java

package com.dinodevs.greatfitwatchface.theme.bin;

import java.util.HashMap;
import java.util.Map;

public class Distance {

    private Number number;
    private Integer suffixImageIndex;
    private Integer decimalPointImageIndex;
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    public Number getNumber() {
        return number;
    }

    public void setNumber(Number number) {
        this.number = number;
    }

    public Integer getSuffixImageIndex() {
        return suffixImageIndex;
    }

    public void setSuffixImageIndex(Integer suffixImageIndex) {
        this.suffixImageIndex = suffixImageIndex;
    }

    public Integer getDecimalPointImageIndex() {
        return decimalPointImageIndex;
    }

    public void setDecimalPointImageIndex(Integer decimalPointImageIndex) {
        this.decimalPointImageIndex = decimalPointImageIndex;
    }

    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: app/src/main/java/com/dinodevs/greatfitwatchface/theme/bin/Distance.kt
package com.dinodevs.greatfitwatchface.theme.bin

import java.util.*

class Distance {
    var number: Number? = null
    var suffixImageIndex: Int? = null
    var decimalPointImageIndex: Int? = null
    private val additionalProperties: MutableMap<String, Any> = HashMap()

    fun getAdditionalProperties(): Map<String, Any> {
        return additionalProperties
    }

    fun setAdditionalProperty(name: String, value: Any) {
        additionalProperties[name] = value
    }
}